quadkey/sf/doc/REFERENCE.md

```diff
@@ -1,3 +1,357 @@
+-- 4356
+-- 4868
+-- 4388
+-- 4900
+```
+
+### SIBLING
+
+{{% bannerNote type="code" %}}
+QUADKEY.SIBLING(quadint, direction)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns the quadint directly next to the given quadint at the same zoom level. The direction must be sent as argument and currently only horizontal/vertical movements are allowed.
+
+* `quadint`: `BIGINT` quadint to get the sibling from.
+* `direction`: `STRING` <code>'right'|'left'|'up'|'down'</code> direction to move in to extract the next sibling. 
+
+**Return type**
+
+`BIGINT`
+
+**Example**
+
+```sql
+SELECT SFCARTO.QUADKEY.SIBLING(4388, 'up');
+-- 3876
+```
+
+### KRING
+
+{{% bannerNote type="code" %}}
+QUADKEY.KRING(quadint, distance)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns an array containing all the quadints directly next to the given quadint at the same level of zoom. Diagonal, horizontal and vertical nearby quadints plus the current quadint are considered, so KRING always returns `(distance*2 + 1)^2` quadints.
+
+* `quadint`: `BIGINT` quadint to get the KRING from.
+* `distance`: `INT` distance (in cells) to the source.
+
+**Return type**
+
+`ARRAY`
+
+**Example**
+
+```sql
+SELECT SFCARTO.QUADKEY.KRING(4388, 1);
+-- 3844
+-- 3876
+-- 3908
+-- 4356
+-- 4388
+-- 4420
+-- 4868
+-- 4900
+-- 4932
+```
+
+### BBOX
+
+{{% bannerNote type="code" %}}
+QUADKEY.BBOX(quadint)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns an array with the boundary box of a given quadint. This boundary box contains the minimum and maximum longitude and latitude. The output format is [West-South, East-North] or [min long, min lat, max long, max lat].
+
+* `quadint`: `BIGINT` quadint to get the bbox from.
+
+**Return type**
+
+`ARRAY`
+
+**Example**
+
+```sql
+SELECT SFCARTO.QUADKEY.BBOX(4388);
+-- 22.5
+-- -21.943045533438177
+-- 45.0
+-- 0.0
+```
+
+### ST_ASQUADINT
+
+{{% bannerNote type="code" %}}
+QUADKEY.ST_ASQUADINT(point, resolution)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns the quadint of a given point at a given level of detail.
+
+* `point`: `GEOGRAPHY` point to get the quadint from.
+* `resolution`: `INT` level of detail or zoom.
+
+**Return type**
+
+`BIGINT`
+
+**Example**
+
+```sql
+SELECT SFCARTO.QUADKEY.ST_ASQUADINT(ST_POINT(40.4168, -3.7038), 4);
+-- 4388
+```
+
+### ST_ASQUADINT_POLYFILL
+
+{{% bannerNote type="code" %}}
+QUADKEY.ST_ASQUADINT_POLYFILL(geography, resolution)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns an array of quadints that intersect with the given geography at a given level of detail.
+
+* `geography`: `GEOGRAPHY` geography to extract the quadints from.
+* `resolution`: `INT` level of detail or zoom.
+
+**Return type**
+
+`ARRAY`
+
+**Example**
+
+```sql
+SELECT SFCARTO.QUADKEY.ST_ASQUADINT_POLYFILL(ST_MAKEPOLYGON(TO_GEOGRAPHY('LINESTRING(-3.71219873428345 40.4133653490709, -3.71440887451172 40.4096566128639, -3.70659828186035 40.4095259047756, -3.71219873428345 40.4133653490709)')), 17);
```

Comment:
Could you update also the bqcarto example using WKT too? Thanks

Reply:
sure!